lib/phlex/sgml.rb

```diff
@@ -406,8 +406,10 @@ def __build_attributes__(attributes, buffer:)
 							end
 						}, buffer: buffer
 					)
+				when Array
```

Comment:
```suggestion
				when Array, Set
```

@joeldrapper can you think of any negatives to also supporting `Set` here?

Reply:
Looks like `Set#join` was added in Ruby 3.0 😕

Do you want to revert this, or just let it go untested and note that Sets are only allowed on Ruby >=3?